vividus-plugin-web-app/src/test/java/org/vividus/steps/ui/web/CodeStepsTests.java

```diff
@@ -88,10 +74,10 @@
     private static final String VARIABLE_NAME = "variableName";
 
     @Mock
-    private ISoftAssert softAssert;
+    private SoftAssert softAssert;
```

Comment:
what is the reason of this change?

Reply:
Follow common practice

Reply:
I changed the interfaces in step classes and obviously Mockito unable to inject interface

Reply:
https://github.com/vividus-framework/vividus/pull/3139